lib/screens/login/login2.dart

```diff
@@ -6,6 +6,10 @@ import '../../models/user.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 
 class LoginFirebaseScreen extends StatefulWidget {
+  final String message;
```

Comment:
As it's only used in the snack bar and we may need this at other places as well. Let's call it something more appropriate. What you think about `displayText` or `snackbarMessage`. Feel free to use your own words, something unique and self-descriptive.